src/utils/number.ts

```diff
@@ -27,14 +29,29 @@ export const parseHashRate = (value: number) => {
   return `${value.toFixed(2)} H/s`
 }
 
-export const parseNumber = (value: any) => {
+export const parseNumber = (value: any, radix?: number) => {
   if (typeof value === 'number') {
     return value
   }
   if (typeof value === 'string') {
+    if (radix) {
+      return parseInt(value, radix)
+    }
     return parseFloat(value)
   }
-  return 0
+  return NaN
+}
+
+export const localeNumberString = (value: any, radix?: number) => {
+  let text = parseNumber(value, radix).toString()
+  if (text === 'NaN') return text
+  const pointIndex = text.indexOf('.')
+  let offset = pointIndex === -1 ? text.length : pointIndex
+  while (offset > 3) {
```

Comment:
I suggest that we can use `. toLocaleString()` to integer part and concat point and decimal part. 

I think use loop to implement separating is not best method.

Reply:
In other locales, I am not sure if the result of `toLocaleString()` meets the requirements.

Reply:
In other locales, I am not sure if the result of toLocaleString() meets the requirements.

Such as Spanish
¥ 1,234.56 -> 1.234,56 CNY